_pages/mobile-interview-project.md

```diff
@@ -94,7 +94,9 @@ There is no pagination API - the endpoint returns the full list of employees. Th
 
 ## Use What You Know
 
-We’re looking to see you at your best – as such, please use the language and frameworks you’re most comfortable with on your chosen mobile platform. This means that if you’re most proficient with Objective-C or Java, use that – don’t feel pressured to use Swift or Kotlin! Square ships all four in production, so we’re not going to prefer answers in one language over another.
+We’re looking to see you at your best – as such, please use the language and frameworks you’re most comfortable with on your chosen mobile platform. This means that if you’re most proficient with Objective-C or Java, use that – don’t feel pressured to use Swift or Kotlin!
+
+Similarly, don’t feel pressured to use SwiftUI, Compose, and/or other libraries and techniques that you are unfamiliar with. We’re not going to prefer answers in one language over another, but we will expect you to be comfortable and proficient with your choices.
```

Comment:
This has been a large stumbling block for a number of candidates in the pairing sessions that stem from the take-home assignment: they set a high level of ambition for showing off the latest and greatest technologies, but they don't understand the more subtle details of these techniques. Case in point: compose - in a pairing context we've seen this backfire a bunch of times - so it seems worth it to us to give a more pointed hint here that we'll have an expectation around proficiency with chosen tools, techniques and languages.

Reply:
💯 .  I know that there was some discussion in slack the other day about the fact that this is kind of a test in itself for candidates, but I think there is something to be said here about us (Square) being viewed by many in the mobile community as intimidating because of our history of open source work, many of the engineers working here and alumni, etc.   I can empathize with candidates who must feel immense pressure to use the latest and greatest libraries and frameworks.  Especially, if they come from a company writing Android and iOS but with older tools and techniques.  These candidates would feel even more pressure.  I think it's warranted to mention this and I think the key part is:

```
We’re not going to prefer answers in one language over another, but we will expect you to be comfortable and proficient with your choices.
```

This is very obvious now what we want and why.  If the candidate still ignores this instruction, then it likely will not work out for them, but if they do.  We may find a very talented engineer who can fit into our eng culture, that we may have dismissed otherwise.

My 2 cents :-)

Reply:
This is a good idea. Maybe we can be even more explicit too by adding to the end that we'll be building on top of these in the on-site so being comfortable with them is better.

Reply:
I like that - that's a +1 i think the current "https://square.github.io/microsite/mobile-interview-project/" version does that we are lacking here. What if we incorporate this from the current version: 


🏢 Onsite Interview Additions
When you come in for your virtual onsite interview, you will work with two engineers over the course of two pair programming sessions (1 hour each) to add more functionality to your application. In the first pair programming session, we’ll focus on extending the data model. In the second pair programming, we’ll focus on extending the UI.

During the interview, the interviewer will also ask you about general architecture choices you made while writing the app. Please be prepared to answer these questions as they come up!
```